### PR TITLE
Documentation: simplify docker-compose.portainer.yml

### PR DESCRIPTION
**Pull request by ftibi93**
## Proposed change

Replaced current `USERMAP_GID` value of 100 with 1000 to match the [docker-compose.env](https://github.com/paperless-ngx/paperless-ngx/blob/36ecb8587d418ba1c915dd857a94c162156c701a/docker/compose/docker-compose.env#L5) file and the [documentation](https://github.com/paperless-ngx/paperless-ngx/blob/36ecb8587d418ba1c915dd857a94c162156c701a/docs/setup.md?plain=1#L153). 

## Type of change

- [ ] Bug fix: non-breaking change which fixes an issue.
- [ ] New feature / Enhancement: non-breaking change which adds functionality. _Please read the important note above._
- [ ] Breaking change: fix or feature that would cause existing functionality to not work as expected.
- [ ] Documentation only.
- [x] Other. Please explain: the current value doesn't really cause any issues, this change just makes it more consistent with the mentioned places.

## Checklist:

- [x] I have read & agree with the [contributing guidelines](https://github.com/paperless-ngx/paperless-ngx/blob/main/CONTRIBUTING.md).
- [ ] If applicable, I have included testing coverage for new code in this PR, for [backend](https://docs.paperless-ngx.com/development/#testing) and / or [front-end](https://docs.paperless-ngx.com/development/#testing-and-code-style) changes.
- [ ] If applicable, I have tested my code for new features & regressions on both mobile & desktop devices, using the latest version of major browsers.
- [ ] If applicable, I have checked that all tests pass, see [documentation](https://docs.paperless-ngx.com/development/#back-end-development).
- [ ] I have run all `pre-commit` hooks, see [documentation](https://docs.paperless-ngx.com/development/#code-formatting-with-pre-commit-hooks).
- [ ] I have made corresponding changes to the documentation as needed.
- [x] I have checked my modifications for any breaking changes.

### CODE REVIEW

**Comment by shamoon** [comment]
Do they even need to be there at all?

**Comment by ftibi93** [comment]
> Do they even need to be there at all?

Nope. I cleaned up the file.

File: docker/compose/docker-compose.portainer.yml [hunk 1]
```diff
@@ -61,28 +63,8 @@ services:
...
-      #PAPERLESS_OCR_LANGUAGES: tur ces
-# Adjust this key if you plan to make paperless available publicly. It should
-# be a very long sequence of random characters. You don't need to remember it.
-      #PAPERLESS_SECRET_KEY: change-me
-# Use this variable to set a timezone for the Paperless Docker containers. If not specified, defaults to UTC.
-      #PAPERLESS_TIME_ZONE: America/Los_Angeles
-# The default language to use for OCR. Set this to the language most of your
-# documents are written in.
-      #PAPERLESS_OCR_LANGUAGE: eng
+    env_file:
```
Comment (shamoon) [inline docker/compose/docker-compose.portainer.yml:66]: Is this necessary in portainer if you upload the env as described?

Comment (ftibi93) [reply docker/compose/docker-compose.portainer.yml:66]: Yes, it is necessary: https://docs.portainer.io/user/docker/stacks/add#option-1-web-editor

**Review by shamoon — APPROVED** [review]

**Pull request merged**